src/prevector.h

```diff
@@ -194,16 +194,29 @@ class prevector {
     T* item_ptr(difference_type pos) { return is_direct() ? direct_ptr(pos) : indirect_ptr(pos); }
     const T* item_ptr(difference_type pos) const { return is_direct() ? direct_ptr(pos) : indirect_ptr(pos); }
 
+    void fill(T* dst, size_type count, const T& value) {
+        for (size_type i = 0; i < count; ++i) {
```

Comment:
nit: `i++` instead of  `++i`

Reply:
The style guide prefers ++i (it doesn't matter for simple types, but for more complex types the prefix version is sometimes more efficient).

Reply:
TIL